.github/workflows/dep_build.yml

```diff
@@ -17,7 +17,9 @@ jobs:
       matrix:
         java_version: ['8', '17', '21']
         # Versions need to align with ones in 'main.yml' workflow
-        kotlin_version: ['1.7.22', '1.8.10', '1.9.23', '2.0.0-Beta4']
```

Comment:
Is there an issue/PR to link to, wrt dropping support for Kotlin 1.7?

I'll make a minor note on https://github.com/FasterXML/jackson/wiki/Jackson-Release-2.18 but could use a link from there.

Reply:
Specifically, this PR is for changing the default Kotlin version; I can link to this one but if so should explain  the reason for dropping in description, if so.

Reply:
This is because `Kotlin 2.0` has been released.
The description has also been modified.

Reply:
Ok but neither title nor description mentions dropping 1.7 -- I can modify title.